comp/process/bundle_test.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package process
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/fx"
+
+	"github.com/DataDog/datadog-agent/comp/process/containercheck"
+	"github.com/DataDog/datadog-agent/comp/process/processcheck"
+	"github.com/DataDog/datadog-agent/comp/process/runner"
+	"github.com/DataDog/datadog-agent/comp/process/submitter"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+func TestBundleDependencies(t *testing.T) {
+	require.NoError(t, fx.ValidateApp(
+		// instantiate all of the process components, since this is not done
+		// automatically.
+		fx.Invoke(func(r runner.Component) {}),
+		fx.Invoke(func(r submitter.Component) {}),
+		fx.Invoke(func(r processcheck.Component) {}),
+		fx.Invoke(func(r containercheck.Component) {}),
+
+		Bundle))
+}
+
+func TestMockBundleDependencies(t *testing.T) {
```

Comment:
What is the difference between `TestBundleDependencies` and `TestMockBundleDependencies`?

Reply:
These are mostly copied from the core bundle at this point, I think we are making sure we can instantiate all mock dependencies in this test. @ogaca-dd could you please clarify intention of these?

Reply:
Thank you!